### PR TITLE
Course

### PR DESCRIPTION
**Pull request by catalin-rucareanu2001**
# Prerequisite Checklist

- [x] Read the [contribution guidelines](https://github.com/open-education-hub/oer-template/blob/main/CONTRIBUTING.md#pull-requests) regarding submitting new changes to the project;
- [x] Tested your changes against relevant architectures and platforms;
- [x] Updated relevant documentation (if needed).

### CODE REVIEW

File: slides/courses/2/number.tex [hunk 1]
```diff
@@ -200,7 +200,7 @@ \subsection{Floating Point}
...
+  The Morris floating-point system is defined by the size of the mantissa and the size of the exponent, with the exponent size referred to as $g$. The first $g$ bits represent the value $G$, which is added with $1$ to obtain the exponent size ($es$). The next bit is the exponent sign bit, followed by $es$ bits, which represent the absolute value of the exponent. The next bit is the fraction sign, and the remaining bits are considered fraction bits. The hidden bit is always $1$. The value is computed by $2^{\text{exponent}} \times (-1)^{\text{fractionSign}} \times (1+\frac{f}{2^{fs}})$, where exponent is $(-1)^{\text{exponentSign}} \times \text{exponentAbsoluteValue}$. The $0$ is represented by all bits of zero, and error cases are represented by only $1s$
```
[the remark this answers is not in the document]
Comment (sdcioc) [reply slides/courses/2/number.tex:203]: It is a nice two page paper [Tapered Floating Point](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=1671767&casa_token=J1qw5rpHcdcAAAAA:OLE_qygMYYBRVknqlY7lHP6Z9ftGAu0fPjtJbJombZlNAtCCiySZk9TAfDi4y8-8MCZYwjOWWI0Lbg&tag=1)

[the remark this answers is not in the document]
Comment (catalin-rucareanu2001) [reply slides/courses/2/number.tex:203]: ok, i'll set it back to its original version

**Review by sdcioc — APPROVED** [review]

**Review by catalin-rucareanu2001 — COMMENTED** [review]
Looks good

File: slides/courses/4/number.tex [hunk 2]
```diff
@@ -215,7 +216,8 @@ \subsection{MUL}
 
 \begin{frame}
     \frametitle{Booth's Algorithm}
- Booth's algorithm is a multiplication algorithm that multiplies two signed binary numbers in two-component notation. The algorithm reduces the number of additions required for the multiplication of two numbers.
+ Booth's algorithm is a multiplication algorithm that multiplies two signed binary numbers in two-component notation.
```
Comment (catalin-rucareanu2001) [inline slides/courses/4/number.tex:219]: multiplication algorithm that multiplies two signed binary numbers in two-component notation.

Shouldn't it be two's-complement? Instead

**Review by catalin-rucareanu2001 — COMMENTED** [review]
One comment one question outside which is oll korrect

**Review by sdcioc — APPROVED** [review]

**Pull request merged**